extensions/sql-database-projects/src/dialogs/insertSqlBindingQuickpick.ts

```diff
@@ -0,0 +1,76 @@
+import * as vscode from 'vscode';
+import * as constants from '../common/constants';
+
+export async function launchInsertSqlBindingQuickpick(uri: vscode.Uri): Promise<void> {
+	if (!uri) {
```

Comment:
param should be `vscode.Uri | undefined` then if this is a valid scenario (same with the command callback in mainController.ts)